test/datahike_server/integration_test.clj

```diff
@@ -150,11 +139,40 @@
 
 (deftest schema-test
   (testing "Fetches current schema"
-    (is (= #:db{:ident #:db{:unique :db.unique/identity}
-                :db.entity/attrs #:db{:cardinality :db.cardinality/many}
-                :db.entity/preds #:db{:cardinality :db.cardinality/many}
-                :db/txInstant {:db/noHistory true}}
+    (add-test-schema)
+    (is (= {:name #:db{:ident       :name,
+                       :valueType   :db.type/string,
+                       :cardinality :db.cardinality/one,
+                       :id          1}}
            (api-request :get "/schema"
                         {}
+                        {:headers {:authorization "token neverusethisaspassword"
+                                   :db-name       "users"}})))))
+
+(deftest reverse-schema-test
+  (testing "Fetches current schema"
```

Comment:
Should be "Fetches current reverse schema" or similar

Reply:
Will fix that.

Reply:
fixed it.